q2cli/tools.py

```diff
@@ -37,14 +37,36 @@ def export_data(path, output_dir):
     result.export_data(output_dir)
 
 
+def show_importable_types(ctx, param, value):
+
```

Comment:
Since there's only a handful of items it probably doesn't hurt to include all of them in the assertions (like you had in that commit). Thanks!